dmscripts/generate_database_data.py

```diff
@@ -121,7 +121,17 @@ def set_all_frameworks_to_expired(data: DataAPIClient) -> None:
             data.update_framework(framework_slug=framework["slug"], data={"status": "expired"})
 
 
-def add_live_g_cloud_framework(data: DataAPIClient) -> None:
+def make_gcloud_12_live(data: DataAPIClient) -> None:
+    data.update_framework(
+        framework_slug=G_CLOUD_FRAMEWORK["slug"],
+        data={
+            "status": "live",
+            "clarificationQuestionsOpen": False
```

Comment:
As per https://alphagov.github.io/digitalmarketplace-manual/content-and-frameworks/framework-lifecycle-for-developers.html#id6, I don't think we need to disable clarification questions when we make the framework live.

Reply:
You are right. Fixed in https://github.com/alphagov/digitalmarketplace-scripts/pull/707/commits/773a71f9a5f23401c72e5f7486e93d56261db776